packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss

```diff
@@ -55,6 +55,9 @@ $isPaddedMargin: 24px;
     position: relative;
     padding: 0;
     margin: 0;
+    display: inline-flex;
+    justify-content: center;
+    flex-direction: column;
```

Comment:
Nitpick: You could probably just use `align-items: center;` to center without changing the direction here.